lib/portlayer/storage/nfs/volume.go

```diff
@@ -109,6 +110,11 @@ func (v *VolumeStore) VolumeCreate(op trace.Operation, ID string, store *url.URL
 	}
 
 	u, _ := v.Service.URL()
+	if u.Scheme != "nfs" {
```

Comment:
@matthewavery This really shouldn't be here at all.  See previous review comment about moving that to the Service so this is not needed.